neurons/validator.py

```diff
@@ -364,15 +364,38 @@ def clean_models(self):
         while not self.stop_event.is_set():
             try:
                 bt.logging.trace("Starting cleanup of stale models.")
-                # Clean out unreferenced models older than 5 mintues.
+                # Clean out models that will not be evaluated older than 5 minutes.
```

Comment:
Technically we do not need a grace period any more. Could remove entirely but also we would catch it on the next one.